open_spiel/python/algorithms/stackelberg_lp_test.py

```diff
@@ -32,12 +32,19 @@
 commit_strategy1 = np.array([1 / 3, 2 / 3])
 commit_value1 = 4 / 3
 
+# a game with dominated strategy
+game2 = pyspiel.create_matrix_game([[3, 9], [9, 1],
+                                    [0, 0], [1, 8]])
```

Comment:
You need the payoffs for the other player(s).

(Or, if it's a two-player zero-sum game, feel free to add a create_two_player_zero_sum_matrix_game that takes only the first player's payoffs)

Reply:
Wait, did you mean:

```
game2 = pyspiel.create_matrix_game([[3, 9], [9, 1]],
                                    [[0, 0], [1, 8]])
```

?

Reply:
oh yes it should be [3, 9], [9, 1]], [[0, 0], [1, 8]]. I just fixed that.